webapp/components/interop-summary.js

```diff
@@ -221,7 +221,13 @@ class InteropSummary extends PolymerElement {
     this.updateSummaryScore(scoreNumbers[0], scores[scores.length - 1][summaryFeatureName]);
     // Update the rest of the browser scores.
     for (let i = 1; i < scoreNumbers.length; i++) {
-      this.updateSummaryScore(scoreNumbers[i], scores[i - 1][summaryFeatureName]);
+      let browserScore = scores[i - 1][summaryFeatureName];
+      // For Interop 2022, 10% of the browser score was based on the investigation scores.
+      if (this.year === '2022') {
+        const investigationScore = this.dataManager.getYearProp('investigationTotalScore');
```

Comment:
Just curious: How hard would it be to modify the CSV to hard code the new browser score so we don't need to calculate it?

While we are committing this, we are still technically generating the browser score on every load.

Reply:
If it would be too much effort, I could be convinced to keep this. I just wanted to weigh the options.
On one hand, I would rather not get in the habit of one-off calculations per year for these frozen scores.
On the other hand, we could always revisit this when we freeze next year's score. And see if this will also apply then.

Reply:
> Just curious: How hard would it be to modify the CSV to hard code the new browser score so we don't need to calculate it?

It's possible, but would require writing new logic to read final numbers from new locations. I'm not opposed to it, but  I feel like there is not a lot of gain for us to do so, since the user experience would be mostly the same, and the calculations are on the client side and inexpensive.

> we could always revisit this when we freeze next year's score.

I think this might be the best path forward. There is talk of spinning off previous years into separate files, which might make this approach of having final numbers already calculated a bit easier. Perhaps we can create an issue to address this when the preparations for next year roll around?

Reply:
Ahh. Thanks for answering that.

And yeah create the issue / add this to an existing issue.